### DISCUSSION TITLE
Allow subscribing to additional filter lists

### DISCUSSION THREAD

**Discussion by james246** [Barre/privaxy#17] [Ideas]
uBlock has a nice feature where you can import custom lists that you provide via a URL, which are also automatically updated. Might be a nice addition to privaxy?

**Comment by Barre** [comment]
I initially wanted to implement this but ultimately did not because I felt that lists that are good enough to be fully included through a feature like this could just as well be contributed to the repo.

Would you have an example where the above reasoning could hardly apply?

**Reply by james246** [reply]
I suppose my use-case is quite niche, but I maintain my own block list (just a `txt` file that I host) so I can subscribe to it on multiple devices (uBlock for Firefox on desktop + Android, iOS Adguard, OpenWRT, etc) which means I don't have to update each device's custom rules individually.

For the less technical user, subscribing to a block list via privaxy's UI might be easier and frictionless than contributing the repo?

I use some of the lists maintained [here](https://github.com/EnergizedProtection/block), if that could be considered for inclusion in the repo?
